src/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="../css/styles.css" type="text/css">
+</head>
+<body>
+    <header>
+        <img src="../src/assets/images/header-shapes-mobile@2x.svg" alt="background" class="bg_shape">
+        <nav>
+            <ul class="flex-nav">
+                <li class="logo">
+                    <a href=""><img alt="logo" src="../src/assets/images/my_logo.png"></a>
+                </li>
+                <li class="menu">
+                    <a href=""><img alt="Menu" src="../src/assets/images/img_icon.png"></a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <section class="wrap">
+        <h1>I am TEEJAY! Happy to have here today!!</h1>
```

Comment:
- Please, kindly use the correct font family, font weight, and font size as used n the Figma design to match the design as much as possible.